client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx

```diff
@@ -245,7 +245,9 @@ const DomainsStep: Step = function DomainsStep( { navigation, flow } ) {
 		}
 
 		if ( flow === DOMAIN_UPSELL_FLOW ) {
-			return exitFlow?.( `/setup/${ flowToReturnTo }/launchpad?siteSlug=${ siteSlug }` );
+			return submit?.( {
+				exitFlowTo: `/setup/${ flowToReturnTo }/launchpad?siteSlug=${ siteSlug }`,
+			} );
```

Comment:
This doesn't add much value, the step is still making decisions based on the flow. And exiting the flow. 

[Now that `goBack` isn't deprecated anymore](https://github.com/Automattic/wp-calypso/pull/77072), feel free to just call it. And implement it in the `DOMAIN_UPSELL_FLOW` flow. 

LMK if this is unclear, we can do a quick huddle.

Reply:
@alshakero - Makes sense. I'll update this to use goBack instead and move the launchpad url into the domain_upsell flow file.